dataprofiler/profilers/profile_builder.py

```diff
@@ -616,11 +641,22 @@ def _update_profile_from_chunk(df, profile=None, sample_size=None,
             def tqdm(l):
                 for i, e in enumerate(l):
                     print("Processing Column {}/{}".format(i+1, len(l)))
-                    yield e
-
+                    yield e                    
 
         # Shuffle indices ones and share with columns
         sample_ids = [*utils.shuffle_in_chunks(len(df), len(df))]
+
+        pool = None
+        if options.structured_options.multiprocess.is_enabled:
+            cpu_count = 1
+            try:
+                cpu_count = mp.cpu_count()
+            except NotImplementedError as e:
```

Comment:
why would it raise a `NotImplementedError`?

Reply:
https://docs.python.org/3/library/multiprocessing.html#multiprocessing.cpu_count